dotCMS/src/main/java/com/dotcms/experiments/business/web/ExperimentWebAPIImpl.java

```diff
@@ -0,0 +1,191 @@
+package com.dotcms.experiments.business.web;
+
+import static com.dotcms.util.CollectionsUtils.list;
+import static com.dotcms.util.CollectionsUtils.map;
+import static org.junit.Assert.assertEquals;
+
+import com.dotcms.experiments.model.Experiment;
+import com.dotcms.experiments.model.ExperimentVariant;
+import com.dotcms.experiments.model.TrafficProportion;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.htmlpageasset.model.HTMLPageAsset;
+import com.dotmarketing.portlets.rules.model.Rule;
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.UtilMethods;
+import com.liferay.util.StringPool;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import net.bytebuddy.utility.RandomString;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Default implementation of {@link ExperimentWebAPI}
+ */
+public class ExperimentWebAPIImpl implements ExperimentWebAPI {
+
+    @Override
+    public List<SelectedExperiment> isUserIncluded(final HttpServletRequest request, final HttpServletResponse response)
+            throws DotDataException, DotSecurityException {
+        final List<Experiment> experimentRunning = APILocator.getExperimentsAPI()
+                .getRunningExperiment();
+
+        final List<Experiment> experiments = pickExperiments(experimentRunning, request, response);
+
+        if (!experiments.isEmpty()) {
+            return  experiments.stream()
+                        .map(experiment -> createSelectedExperimentAndCreateCookie(response,
+                                experiment))
+                        .collect(Collectors.toList());
+        } else {
+            setCookie(NONE_EXPERIMENT, response, getDefaultExpireDate());
+            return list(NONE_EXPERIMENT);
+        }
+    }
+
+    @NotNull
```

Comment:
remove this

Reply:
done https://github.com/dotCMS/core/pull/23334